crates/ty_python_semantic/resources/mdtest/pep695_type_aliases.md

```diff
@@ -244,6 +244,16 @@ def f(x: IntOr, y: OrInt):
         reveal_type(x)  # revealed: Never
     if not isinstance(y, int):
         reveal_type(y)  # revealed: Never
+
+# TODO emit a diagnostic on this line
+type Itself = Itself
+
+def foo(Itself: Itself):
+    x: Itself
```

Comment:
IMO it's a bit odd to test that you can annotate a parameter as `Itself`, and then use that parameter again as an annotation. It may work due to how we treat `Unknown` in a type annotation, but I don't think this is something we want to commit to in a test.

I would probably just have done this instead:

```suggestion
def foo(x: Itself):
    reveal_type(x)  # Unknown
```

Is there a particular reason why you think it's important to test this "annotated value as annotation" behavior?